backend/src/services/course.service.ts

```diff
@@ -438,8 +402,8 @@ const createCourse = async (req: RequestHasLogin): Promise<ResponseBase> => {
         });
 
         if (uploadFileResult) {
```

Comment:
File này có 1 số problem sau : 
- nest if
- constant nếu nó nhiều quá nên cho nó vô 1 dối tượng vd MESSAGE.MESSAGE_SUCCESS_GET_DATA
- foreach lồng nhau
- sử dụng object property shorthand của es6
- chỗ newCategories khai báo mảng mới xong for categories để làm gìhttps://github.com/cvn-intern/Freshemy/blob/9828253a77f38016cf317b805b52f140faf87e23/backend/src/services/course.service.ts#L265-L267
- Bên trong class này a thấy logic khá phức tạp , nên vận dụng kiến thức es6 vs clean code để đơn giản hóa vấn đề lại